assets/scripts/info_bubble/info_bubble.js

```diff
@@ -1,9 +1,5 @@
-/* global app, debug, street, system */
-/* global SEGMENT_WARNING_WIDTH_TOO_LARGE,
-      KEYS, SEGMENT_WARNING_OUTSIDE,
-      SEGMENT_WARNING_WIDTH_TOO_SMALL */
-/* global _saveStreetToServerIfNecessary, _processWidthInput */
-// Many things in resizing.js
+/* global app, debug, system, KEYS,
+   _processWidthInput */
```

Comment:
`_processWidthInput` is not global - it should be imported from `../util/width_units`